doc/dev/fluxing.rst

```diff
@@ -0,0 +1,116 @@
+.. include:: ../include/links.rst
+
+.. _fluxcalib:
+
+Flux Calibration
+=================================
+
+Version History
+---------------
+
+=========   =============   =========== ===========
+*Version*   *Author*        *Date*      ``PypeIt``
+=========   =============   =========== ===========
+1.0         Joe Hennawi     25 Jan 2021 1.1.1
+=========   =============   =========== ===========
```

Comment:
Why the revision history? I haven't seen that in any of the other documents we have. Is this something new we're doing?

Reply:
I think I got this from the DEIMOS document that @kbwestfall created. That is what I copied to get started.